optimizer/plan/plan_test.go

```diff
@@ -0,0 +1,383 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plan
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ngaut/log"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/ast"
+	"github.com/pingcap/tidb/model"
+	"github.com/pingcap/tidb/parser"
+)
+
+var _ = Suite(&testPlanSuite{})
+
+func TestT(t *testing.T) {
+	TestingT(t)
+}
+
+type testPlanSuite struct{}
+
+func (s *testPlanSuite) TestRangeBuilder(c *C) {
+	log.SetLevel(log.LOG_LEVEL_ERROR)
```

Comment:
The name LOG_LEVEL_ERROR doesn't fit our coding style now, need fixing it?
@ngaut